### ISSUE TITLE
Interpolation brace style

### ISSUE THREAD

**Issue by blaumeise20** [atom/language-php#454]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

I am comming from the VS Code repository, and I have some problems with syntax highlighting. When writing this code:
```php
<?php
$example = "example";
$somevar = "{$example}_{$example}";
echo $somevar;
```
Most things are highlighted correctly, but the `{}` symbols don't look right.

![Bildschirmfoto 2022-04-22 um 15 34 19](https://user-images.githubusercontent.com/62756994/164724858-9d3c31bc-90f3-4353-9442-8e35b69bfb4a.png)

### Steps to Reproduce

Copy the above code snippet into an editor using this repository's syntax for highlighting.

**Expected behavior:**

The braces should be blue like in most other languages.

Example JavaScript:
![Bildschirmfoto 2022-04-22 um 15 36 34](https://user-images.githubusercontent.com/62756994/164725287-b2d6d665-0222-4504-9471-764057b2f22e.png)

**Actual behavior:**

The braces have string color which looks weird.

**Comment by blaumeise20** [comment]
I tried the configuration you provided, but for some reason it doesn't only change the braces, but also every dollar symbol in the file:
![Bildschirmfoto 2022-04-25 um 07 18 44](https://user-images.githubusercontent.com/62756994/165025140-344abf93-4b30-4dd7-b22c-8792a21a13c3.png)

Also I am using Dark+ (default in VS Code).

**Comment by KapitanOczywisty** [comment]
@blaumeise20 Yes, but I've edited to do this only in strings, and with the correct color (`#9CDCFE`) it doesn't matter. Try edited configuration.

**Comment by blaumeise20** [comment]
Oh, ok, I tried the new configuration, it works better, but still makes the dollar sign blue inside variables (also I want to have that dark blue color like in JS, not the variable color).

**Comment by KapitanOczywisty** [comment]
> but still makes the dollar sign blue inside strings

I don't understand that, shouldn't dollar sign be blue there?

You can make 2 rules, one to make braces colored, second one to restore dollar color:

```
      {
        "scope": [
          "string.quoted.double.php punctuation.definition.variable.php",
          "string.unquoted.heredoc.php punctuation.definition.variable.php"
        ],
        "settings": {
          "foreground": "#569CD6"
        }
      },
      {
        "scope": [
          "string variable punctuation.definition.variable.php",
          "string variable punctuation.definition.variable.php"
        ],
        "settings": {
          "foreground": "#9CDCFE"
        }
      },
```

With `Commands` (`ctrl+shift+p`) -> `Developer: Inspect Editor Tokens and Scopes` you can check names of tokens under the cursor.
